pkg/plugins/backendplugin/coreplugin/registry.go

```diff
@@ -41,6 +43,12 @@ const (
 	Grafana         = "grafana"
 )
 
+func init() {
+	// Non-optimal global solution to replace plugin SDK default loggers for core plugins.
+	sdklog.DefaultLogger = &logWrapper{logger: log.New("plugin.coreplugin")}
```

Comment:
I think this can be simplified as `sdklog.DefaultLogger = sdklog.New("plugin.coreplugin")` so you don't need to define `logWrapper`.

BTW, what happens if something calls backend.Logger and it's not a core plugin?

Reply:
No, that doesn't work. Then we get the behavior we already have since `sdklog.New` creates a https://github.com/hashicorp/go-hclog logger implementation.

> BTW, what happens if something calls backend.Logger and it's not a core plugin?

The log statements are sent to Grafana over gRPC and will be included in Grafana's server log.

Reply:
> No, that doesn't work. Then we get the behavior we already have since `sdklog.New` creates a https://github.com/hashicorp/go-hclog logger implementation.

Gotcha

> > BTW, what happens if something calls backend.Logger and it's not a core plugin?
> 
> The log statements are sent to Grafana over gRPC and will be included in Grafana's server log.

yes, I mean, would those log entries include `logger=plugin.coreplugin` even if it's not logged by a core plugin?